src/models/activityReportObjectiveTopic.js

```diff
@@ -25,11 +25,11 @@ export default (sequelize, DataTypes) => {
       type: DataTypes.INTEGER,
     },
     activityReportObjectiveId: {
-      type: DataTypes.STRING,
```

Comment:
yikes

Reply:
I don't see this in latest code. To confirm it was in the aro topics model?

Reply:
Yikes that it was there not that what you did was wrong